From c333b4ce605470086d2e6e6b6e015cfd845e5b18 Mon Sep 17 00:00:00 2001
From: Billy Laws <blaws05@gmail.com>
Date: Sun, 12 Mar 2023 15:20:14 +0000
Subject: [PATCH 04/12] Remove yuzu specific handling from CMakeLists

---
 CMakeLists.txt | 30 ------------------------------
 1 file changed, 30 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 418eb5c..786b564 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -195,8 +195,6 @@ add_library(audio_core STATIC
     sink/sink_stream.h
 )
 
-create_target_directory_groups(audio_core)
-
 if (MSVC)
     target_compile_options(audio_core PRIVATE
         /we4242 # 'identifier': conversion from 'type1' to 'type2', possible loss of data
@@ -212,31 +210,3 @@ else()
         -Wno-sign-conversion
     )
 endif()
-
-if (ARCHITECTURE_x86_64 OR ARCHITECTURE_arm64)
-    target_link_libraries(audio_core PRIVATE dynarmic::dynarmic)
-endif()
-
-if (ENABLE_CUBEB)
-    target_sources(audio_core PRIVATE
-        sink/cubeb_sink.cpp
-        sink/cubeb_sink.h
-    )
-
-    target_link_libraries(audio_core PRIVATE cubeb::cubeb)
-    target_compile_definitions(audio_core PRIVATE -DHAVE_CUBEB=1)
-endif()
-
-if (ENABLE_SDL2)
-    target_sources(audio_core PRIVATE
-        sink/sdl2_sink.cpp
-        sink/sdl2_sink.h
-    )
-
-    target_link_libraries(audio_core PRIVATE SDL2::SDL2)
-    target_compile_definitions(audio_core PRIVATE HAVE_SDL2)
-endif()
-
-if (YUZU_USE_PRECOMPILED_HEADERS)
-    target_precompile_headers(audio_core PRIVATE precompiled_headers.h)
-endif()
-- 
2.39.0

